src/build/verification.ts

```diff
@@ -88,11 +90,33 @@ export async function verifyNoAdvancedAPIRoutes(ctx: PluginContext) {
   }
 }
 
-export function verifyNoNetlifyForms(ctx: PluginContext, html: string) {
-  if (!warnings.has('netlifyForms') && /<form[^>]*?\s(netlify|data-netlify)[=>\s]/.test(html)) {
+const formDetectionRegex = /<form[^>]*?\s(netlify|data-netlify)[=>\s]/
+
+export async function verifyNetlifyFormsWorkaround(ctx: PluginContext) {
+  const srcDir = ctx.resolveFromSiteDir('public')
+  const paths = await glob(join(srcDir, '**/*.html'))
+  try {
+    const hasWorkaround = await paths.some(async (path): Promise<boolean> => {
```

Comment:
I'm pretty sure this doesn't work as expected - `some` predicate will always be truthy as it always return a promise so `hasWorkaround` would always be `true`?

Reply:
Can you add html file with this snippet (so missing `netlify` attribute) to previous fixture (i.e. `tests/fixtures/netlify-forms/public/not-netlify-form.html`)?:

```html
<form name="contact"></form>
```

in my local testing adding that file make the test we have for actually showing warning no longer do that, because `hasWorkaround` will be truthy as long there are any `.html` files in `public` dir

Reply:
well caught! fixed in [62faedc](https://github.com/netlify/next-runtime/pull/2523/commits/62faedc8ae7a6eae2542b859f175585830f264b2) and test modifications made as suggested